app/Models/Airline.php

```diff
@@ -20,7 +20,7 @@ class Airline extends Model
         'fuel_100ll_cost',
         'fuel_jeta_cost',
         'fuel_mogas_cost',
-        'active',
+        'active'
```

Comment:
Leave the trailing commas on any lists like this. Sticking with the PSR conventions. Some goes for the other lines below